docs/source/release/v3.7.0/diffraction.rst

```diff
@@ -90,6 +90,9 @@ Powder Diffraction Scripts
 - PowderISIS/Aziz script has been renamed to CryPowderISIS and can be found
   `Script/CryPowderISIS`
 
+- :ref:`pearl-powder-diffraction-ref` documentation has been implemented and
+  PowderISIS/Aziz script documentation has been renamed to ref:`cry-powder-diffraction-ref`
```

Comment:
I think that the name Aziz scrip was not used? So I'd simply call it `PowderISIS` scripts.